exercises/binary-search-tree/test/binary_search_tree_tests.erl

```diff
@@ -0,0 +1,112 @@
+%% Based on canonical data version 1.4.0
+%% https://github.com/exercism/problem-specifications/raw/master/exercises/bob/canonical-data.json
+%% This file is automatically generated from the exercises canonical data.
```

Comment:
If the test suite is created manually (ie, there is no test generator), this line is wrong...

Reply:
It were even wrong if it were from the generator, as it claims to be generated from `bob`s canonical data…